mtags/src/main/scala-3/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -176,9 +176,11 @@ abstract class PcCollector[T](driver: InteractiveDriver, params: OffsetParams):
          */
         def isForComprehensionOwner(named: NameTree) =
           soughtNames(named.name) &&
-            named.symbol.owner.isAnonymousFunction && owners.exists(
-              _.span.point == named.symbol.owner.span.point
-            )
+            scala.util
```

Comment:
Is this change needed? Does it solve any other issues?

Reply:
Well, the fix works without this change, but some exception still pops out

Reply:
ok sure, it make sense to add try there, thanks for the explanation!